### PR TITLE
Rain drops game

### PR DESCRIPTION
**Pull request by AlekseyGrimm**
https://github.com/rolling-scopes-school/tasks/blob/master/tasks/stage-0/raindrops.md

### CODE REVIEW

**Comment by y-solovey** [comment]
Do you use the file `RainDrops_Game/sounds/sea.mp3` ?

File: RainDrops_Game/style.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+* {
```
Comment (y-solovey) [inline RainDrops_Game/style.css]: Do we use this file?

File: RainDrops_Game/indexGame.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                <button class="calc-button calc-button-numbers three">3</button>
+                <button id="clear" class="calc-button calc-button-text clear">Clear</button>
+                <button id="enter" class="calc-button calc-button-text enter">Enter</button>
+                <button id="delete" class="calc-button calc-button-text delete">Delete</button>
+                <button class="calc-button calc-button-numbers zero">0</button>  
+            </div>
+        </div>
+        <audio id="sound" src="./sounds/dendi.mp3"></audio>
+        <audio id="soundFalse" src="./sounds/falseAnswer.mp3"></audio>
+        <audio id="soundTrue" src="./sounds/trueAnswer.mp3"></audio>
```
Comment (y-solovey) [inline RainDrops_Game/indexGame.html:54]: where we use **sea.mp3** file?

File: RainDrops_Game/main.js [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+
+function goNextPage() {
+  wrapperFirstPage.style.display = "none";
+  start.style.display = "flex";
+ };
+
+//RANDOM NUMBER AND OPERATOR
+
+function getRandomEquation(min, max) {
+  function getRandomNumber(min, max) {
```
Comment (y-solovey) [inline RainDrops_Game/main.js]: please move functions out of `getRandomEquation`

File: RainDrops_Game/main.js [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+  if (score > 450) {
+    min = 0;
+    max = 15;
+  }
+  const equation = getRandomEquation(min, max);
+  const bonus = dropsCount % 6 === 0;
+  dropsCount++;
+  createDrop(equation, currentId, bonus);
+
+  function removeDrop() {
```
Comment (y-solovey) [inline RainDrops_Game/main.js]: move function out

File: RainDrops_Game/main.js [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+    trueSong.play();
+    trueAnswer++;
+  } else {
+    falseSong.play();
+  }
+};
+
+// ADD ENTER FROM KEYBOARD
+function addKeyBoard(e) {
+  if (result.textContent.length > 5) {
```
Comment (y-solovey) [inline RainDrops_Game/main.js]: why we need this line?

File: RainDrops_Game/indexGame.html [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+                <button class="calc-button calc-button-numbers one">1</button>
+                <button class="calc-button calc-button-numbers two">2</button>
+                <button class="calc-button calc-button-numbers three">3</button>
+                <button id="clear" class="calc-button calc-button-text clear">Clear</button>
+                <button id="enter" class="calc-button calc-button-text enter">Enter</button>
+                <button id="delete" class="calc-button calc-button-text delete">Delete</button>
+                <button class="calc-button calc-button-numbers zero">0</button>  
+            </div>
+        </div>
+        <audio id="sound" src="./sounds/sea.mp3"></audio>
```
Comment (y-solovey) [inline RainDrops_Game/indexGame.html]: should we add `loop` attribute here?

File: RainDrops_Game/main.js [hunk 7]
```diff
@@ -0,0 +1,383 @@
...
+};
+
+function gameOver() {  
+  setTimeout(() => {
+    game_over.style.display = "block";
+    start.style.display = "none";
+    allScore.textContent = score;
+    totalEquations.textContent = dropsCount - 1;
+    totalAnswer.textContent = trueAnswer;
+    song.stop();
```
Comment (y-solovey) [inline RainDrops_Game/main.js]: `audio` does not have `stop` method

File: RainDrops_Game/main.js [hunk 8]
```diff
@@ -0,0 +1,383 @@
...
+    falseSong.play();
+  }
+};
+
+// ADD ENTER FROM KEYBOARD
+function addKeyBoard(e) {
+  if (result.textContent.length > 5) {
+    return result.textContent;
+  }
+  const zero = result.textContent === "0";
```
Comment (y-solovey) [inline RainDrops_Game/main.js]: why we need this line?

**Review by y-solovey — CHANGES_REQUESTED** [review]

File: RainDrops_Game/main.js [hunk 9]
```diff
@@ -0,0 +1,383 @@
...
+  }
+  return (result.textContent = "0");
+};
+
+// CREATE DROP
+
+function createDrop(equation, id, isBonus) {
+  const drop = document.getElementById("drop");
+  let div = document.createElement("div");
+  div.className = "raindrop";
```
Comment (y-solovey) [inline RainDrops_Game/main.js]: please use `div.classList.add("raindrop")`

File: RainDrops_Game/indexGame.html (hunk 2 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/indexGame.html:54]: update this code

File: RainDrops_Game/style.css (hunk 1 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/style.css]: delete

File: RainDrops_Game/main.js (hunk 3 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/main.js]: update this code

File: RainDrops_Game/main.js (hunk 5 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/main.js]: without this function, all pressed numbers are displayed

Comment (AlekseyGrimm) [reply RainDrops_Game/main.js]: shrink the window and see

File: RainDrops_Game/indexGame.html (hunk 6 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/indexGame.html]: I think that it is not

File: RainDrops_Game/main.js (hunk 7 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/main.js]: pardon)))) I'm add pause( )

File: RainDrops_Game/style.css [hunk 10]
```diff
@@ -0,0 +1,47 @@
+body {
```
Comment (y-solovey) [inline RainDrops_Game/style.css]: this file does not included in html

File: RainDrops_Game/main.js (hunk 4 above)
Comment (y-solovey) [reply RainDrops_Game/main.js]: try to create `function removeDrop(currentId, up) {}` and move it up

File: RainDrops_Game/gamePlace.css [hunk 11]
```diff
@@ -0,0 +1,393 @@
...
+
+.calc-button:hover {
+    background-color: rgb(230, 216, 32);
+}
+
+.score {
+    display: flex;
+    height: 10%;
+    width: 90%;
+    transform: translateY(-35%);
```
Comment (y-solovey) [inline RainDrops_Game/gamePlace.css]: why we need this line?

File: RainDrops_Game/main.js [hunk 12]
```diff
@@ -0,0 +1,313 @@
...
+
+function onAutoPlay() {
+  startGame();
+  setTimeout(function autoPlay() {
+    setTimeout(autoEnterResult, 4500);
+    if (arrayResult[0]) {
+      enterNumber();
+    }
+    setTimeout(autoPlay, 5000);
+  }, 2500);
```
Comment (y-solovey) [inline RainDrops_Game/main.js:305]: please add "Back" button

**Review by y-solovey — CHANGES_REQUESTED** [review]

File: RainDrops_Game/main.js [hunk 13]
```diff
@@ -0,0 +1,314 @@
...
+const start = document.getElementById("start");
+const game_over = document.getElementById("game-Over");
+const wrapperFirstPage = document.getElementById("wrapperFirstPage");
+const allScore = document.getElementById("allScore");
+const totalEquations = document.getElementById("totalEquations");
+const totalAnswer = document.getElementById("totalAnswer");
+const waveHeight = document.getElementById("wave-wrapper");
+const raindrop = document.getElementsByClassName("raindrop");
+const autoplay = document.getElementById ('autoplay');
+const playButton = document.getElementById('.control__play-button');
```
Comment (y-solovey) [inline RainDrops_Game/main.js]: why we need this const?

File: RainDrops_Game/main.js (hunk 4 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/main.js]: update this code

File: RainDrops_Game/indexGame.html (hunk 6 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/indexGame.html]: update

File: RainDrops_Game/main.js (hunk 9 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/main.js]: update

File: RainDrops_Game/style.css (hunk 10 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/style.css]: delete this

File: RainDrops_Game/gamePlace.css (hunk 11 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/gamePlace.css]: update code

File: RainDrops_Game/main.js (hunk 12 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/main.js:305]: well done

File: RainDrops_Game/main.js (hunk 8 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/main.js]: I'm fix this code

**Review by y-solovey — CHANGES_REQUESTED** [review]

File: RainDrops_Game/main.js (hunk 13 above)
Comment (AlekseyGrimm) [reply RainDrops_Game/main.js]: delete this const

**Pull request closed without merging**